torch/_prims/__init__.py

```diff
@@ -603,8 +603,18 @@ def _not_impl(*args, **kwargs):
 )
 
 
-def _cbrt_aten(a: torch.Tensor):
-    return pow(a, (1 / 3))
+def _cbrt_aten(a: torch.Tensor) -> Tensor:
+    utils.check(
+        not a.is_complex(),
+        lambda: "cbrt: Complex inputs not supported. Consider calling torch.pow(a, 1.0/3.0)",
+    )
+    # Returns the real cubic root of the number.
+    # Note that if a < 0, pow(a, (1. / 3.)) returns th complex number
+    # exp(1/3 * log(a)) = exp(1/3 * (log(abs(a)) + pi*i)) = cbrt(abs(a)) * e^{pi/3*i}
+    # which is a complex number.
+    # For more info see the section Note in
+    # https://en.cppreference.com/w/cpp/numeric/math/cbrt
+    return torch.where(a >= 0, pow(a, (1.0 / 3.0)), -pow(-a, 1.0 / 3.0))
```

Comment:
How about this implementation using 3 kernels instead of 6, 
```suggestion
    return torch.copysign(torch.pow(a.abs(), 1/3), a)
```

Reply:
Do we have `copysign` in PrimTorch?

Reply:
you are completely right, I'll correct it